.prettierrc.json

```
@@ -0,0 +1,10 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": false,
+  "singleQuote": true,
```

Double would be nicer if possible, more consistent with the rest of nf-core...
```suggestion
  "singleQuote": false,
```

---

Would also help to make the diff a _little_ smaller I think